sdk/tests/js/tests/tex-image-and-sub-image-3d-with-image-bitmap-from-image.js

```diff
@@ -138,7 +117,7 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
 
         // Check a few pixels near the top and bottom and make sure they have
         // the right color.
-        var tolerance = 10;
+        var tolerance = 8;
```

Comment:
Thank you for your comments, Ken. I didn't realize that colorspaceConversion would have impact on the result in this case. Once colorspaceConversion is supported, I will definitely add that to all these test cases.